src/components/EmptyPlaceholder/index.js

```diff
@@ -16,7 +16,7 @@ const EmptyPlaceholder = ({ store: { ui, createEvent }, intl, ...props }) => {
     <Row>
       <Row><img src="/images/empty.svg" alt="empty placeholder" /></Row>
       <Row>
-        <Typography variant="title">
+        <Typography variant="subtitle1">
```

Comment:
This doesn't look like the right mapping. Look at the sizing.
https://material-ui.com/style/typography/

Looks like it should be `title -> h6`